variantlib/validators.py

```diff
@@ -153,3 +158,96 @@ def validate_type(value: Any, expected_type: type) -> None:
     wrong_type = _validate_type(value, expected_type)
     if wrong_type is not None:
         raise ValidationError(f"Expected {expected_type}, got {wrong_type}")
+
+
+def validate_variant_json(data: dict) -> None:
```

Comment:
```suggestion
def validate_variants_json(data: dict) -> None:
```

I think it's clearer when it matches the filename.

Reply:
Agreed